From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tom Levy <tomlevy93@gmail.com>
Date: Sat, 13 Jun 2020 16:36:17 +1200
Subject: partial cherry-pick: Change scoreboard Filter to Plain Old Data in
 ScoreboardService

(partially cherry picked from commit 5e407b6affc60ae2701d970984ac9ef31739c8fc)
---
 webapp/src/Service/ScoreboardService.php | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/webapp/src/Service/ScoreboardService.php b/webapp/src/Service/ScoreboardService.php
index 33bf3304d..755f296df 100644
--- a/webapp/src/Service/ScoreboardService.php
+++ b/webapp/src/Service/ScoreboardService.php
@@ -722,10 +722,10 @@ class ScoreboardService
      */
     public function getFilterValues(Contest $contest, bool $jury): array
     {
-        $filters          = [
+        $filters = [
             'affiliations' => [],
-            'countries' => [],
-            'categories' => [],
+            'countries'    => [],
+            'categories'   => [],
         ];
         $showFlags        = $this->dj->dbconfig_get('show_flags', true);
         $showAffiliations = $this->dj->dbconfig_get('show_affiliations', true);
@@ -902,28 +902,28 @@ class ScoreboardService
         }
 
         if ($filter) {
-            if ($filter->getAffiliations()) {
+            if ($filter->affiliations) {
                 $queryBuilder
                     ->andWhere('t.affilid IN (:affiliations)')
-                    ->setParameter(':affiliations', $filter->getAffiliations());
+                    ->setParameter(':affiliations', $filter->affiliations);
             }
 
-            if ($filter->getCategories()) {
+            if ($filter->categories) {
                 $queryBuilder
                     ->andWhere('t.categoryid IN (:categories)')
-                    ->setParameter(':categories', $filter->getCategories());
+                    ->setParameter(':categories', $filter->categories);
             }
 
-            if ($filter->getCountries()) {
+            if ($filter->countries) {
                 $queryBuilder
                     ->andWhere('ta.country IN (:countries)')
-                    ->setParameter(':countries', $filter->getCountries());
+                    ->setParameter(':countries', $filter->countries);
             }
 
-            if ($filter->getTeams()) {
+            if ($filter->teams) {
                 $queryBuilder
                     ->andWhere('t.teamid IN (:teams)')
-                    ->setParameter(':teams', $filter->getTeams());
+                    ->setParameter(':teams', $filter->teams);
             }
         }
 
-- 
2.20.1

